tests/ui/privacy/privacy1.rs

```diff
@@ -129,9 +129,7 @@ mod foo {
     }
 
     fn test2() {
-        use bar::baz::{foo, bar};
-        //~^ ERROR: module `baz` is private
-        //~| ERROR: module `baz` is private
+        use bar::baz::{foo, bar}; //~ ERROR: module `baz` is private
```

Comment:
I'm realizing that outside of the test suite this was likely previously caught by the generic deduplication logic. I'd noticed the duplicated line in the test and it seemed reasonable to avoid it, but this likely never had user impact.

Reply:
Could you remove this new deduplication?